src/rextester.js

```diff
@@ -0,0 +1,137 @@
+const languageProperties = require('./languageProperties.json');
+const superagent = require('superagent')
+
+function parseCode(code, languageObject, imports) {
+    if (languageObject.classDeclaration) { //Add class declaration and replace {{code}} with the user's code
+        let classDec = languageObject.classDeclaration
+        classDec = classDec.replace("{{code}}", code)
+        code = classDec
+    }
+    let i = 0
+    imports.forEach(element => { //If import already exists in defaults then remove it from the array
```

Comment:
There is an optional argument in forEach to have the index as well.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
This will prevent having to declare that i above.

Also I have a tendency to use for of instead of forEach but I am totally fine with this usecase as it is super readable.